### DISCUSSION TITLE
Cannot remove qwant

### DISCUSSION THREAD

**Discussion by nawka12** [searxng/searxng#3844] [Q&A] [answered]
**Version of SearXNG, commit number if you are using on master branch and stipulate if you forked SearXNG**

2024.9.21+0b3724651

**How did you install SearXNG?**

using searxng-docker

**What happened?**

Cannot remove `qwant` from the engine used. Not an issue with `google` so it should be only a qwant thing.

**How To Reproduce**

Set this in the `settings.yml` file:
```yml
use_default_settings:
  engines:
    remove:
      - qwant
```
**Expected behavior**

`qwant` is removed from the engine used by SearxNG

**Screenshots & Logs**

```
searxng  |   File "/usr/local/searxng/searx/webapp.py", line 1345, in <module>
searxng  |     search_initialize(enable_checker=True, check_network=True, enable_metrics=settings['general']['enable_metrics'])
searxng  |   File "/usr/local/searxng/searx/search/__init__.py", line 33, in initialize
searxng  |     initialize_network(settings_engines, settings['outgoing'])
searxng  |   File "/usr/local/searxng/searx/network/network.py", line 394, in initialize
searxng  |     NETWORKS[engine_name] = NETWORKS[network]
searxng  |                             ~~~~~~~~^^^^^^^^^
searxng  | KeyError: 'qwant'
```
**Additional context**

Qwant always came back with gibberish results(?), that's why I want to disable it.
![image](https://github.com/user-attachments/assets/364e9bb0-565d-46d9-abd2-c7651ce417f7)

**Comment by nawka12** [comment]
Currently using `keep_only` as a workaround.

**Comment by return42** [comment, marked as answer]
```
searxng  |   File "/usr/local/searxng/searx/network/network.py", line 394, in initialize
searxng  |     NETWORKS[engine_name] = NETWORKS[network]
searxng  |                             ~~~~~~~~^^^^^^^^^
searxng  | KeyError: 'qwant'
```

If you remove qwant (web), you remove the qwant network which is used by other qwant engines (images, videos, news --> ``network: qwant``).  I assume your intention is to remove all theses qwatn engines.

https://github.com/searxng/searxng/blob/0b3724651eadade974a01902ca2c376a01325f53/searx/settings.yml#L1467-L1494
